nixos/modules/services/security/hologram-server.nix

```diff
@@ -94,9 +96,15 @@ in {
       after       = [ "network.target" ];
       wantedBy    = [ "multi-user.target" ];
 
-      serviceConfig = {
-        ExecStart = "${pkgs.hologram.bin}/bin/hologram-server --debug --conf ${cfgFile}";
-      };
+      inherit script;
```

Comment:
For my launchd modules I added [`command`](https://github.com/LnL7/nix-darwin/blob/master/modules/launchd/default.nix#L50-L54) which defaults to a writeScript if a script is defined.